### ISSUE TITLE
(Accessibility) Lower-Contrast Dark Theme (also Light?)

### ISSUE THREAD

**Issue by gozzarda** [mastodon/mastodon#38685]
### Pitch

I think it would be beneficial to myself and others with similar vision conditions to have lower contrast or pastel medium dark and light themes.

### Motivation

I previously mentioned this in [this comment](https://github.com/mastodon/mastodon/issues/32325#issuecomment-2469445723) on #32325, but that issue has been closed with the ask to reopen any remaining specific issues.

That issue was closed with mention of providing higher-contrast dark themes for accessibility, but I (and I am sure others) have conditions where high contrast is a problem. Indeed, the original dark mode theme (with a more pastel tone and a grey background) before its replacement was near perfect for my eyes, and the new higher contrast options are worse.

For reference, the class of vision conditions I am hoping to see addressed generally result in blur or duplicate images at low opacity, meaning that if the contrast is too high these are very visible, while light grey text on a dark grey background, or similar, results in the anomalies blending into the background.

I expect it is entirely worth it to have medium light and dark modes to have some options I'm between deep black and searing white.

**Comment by diondiondion** [comment]
Thanks, I'd love to see this in Mastodon; and with the way our new theming system is set up in 4.6, it wouldn't be hard to implement either.

However, we do need to look into how much ongoing maintenance work a new contrast setting would incur for our product designer.

**Comment by eishiya** [comment]
Now that 4.6 has been released with "improv[ed] color contrasts" that employ so many high-saturation background colours that using the web client causes me physical pain, some lower-contrast, less saturated theme options feel even more necessary than before.

Mastodon's insistence on unique branding has led it to high-saturation pure blues and purples, which are some of the worst colours for eye strain. This was an issue when the new branding was adopted some time back, but 4.6 has made it much, much worse by employing high-saturation colours as backgrounds *nearly everywhere*.

Perhaps this new high contrast theme was intended to be optional, but at least on my instance, the only contrast options are "Auto" and "High", changing makes no difference.
